api/pkg/apis/v1alpha1/providers/target/script/script.go

```diff
@@ -394,6 +394,15 @@ func (i *ScriptProvider) Apply(ctx context.Context, deployment model.DeploymentS
 			ret[k] = v
 		}
 	}
+
+	for k, v := range ret {
+		ret[k] = v
```

Comment:
It reassigns the value v back to the ret map at key k, which is redundant.

Reply:
Seems this is the redundant code, please remove it.